includes/compat/subscriptions/class-wc-payment-gateway-wcpay-subscriptions-compat.php

```diff
@@ -162,4 +301,21 @@ public function maybe_render_subscription_payment_method( $payment_method_to_dis
 			return $payment_method_to_display;
 		}
 	}
+
+	/**
+	 * Outputs a select element to be used for the Subscriptions payment meta token selection.
+	 *
+	 * @param WC_Subscription $subscription The subscription object.
+	 * @param string          $field_id     The field_id to add to the select element.
+	 * @param string          $field_value  The field_value to be selected by default.
+	 */
+	public function render_custom_payment_meta_input( $subscription, $field_id, $field_value ) {
+		$tokens = $this->get_user_formatted_tokens_array( $subscription->get_user_id() );
+		echo '<select name="' . esc_attr( $field_id ) . '" id="' . esc_attr( $field_id ) . '">';
+		foreach ( $tokens as $token ) {
```

Comment:
Same comment here about adding a `<option value="">Choose one...</option>` if `$field_value` is empty.

Reply:
Good idea! Added in https://github.com/Automattic/woocommerce-payments/pull/892/commits/2683549de85bec95c1fe0b6440075f874ae22f1f.